pkgs/tools/networking/telepresence2/default.nix

```diff
@@ -1,27 +1,51 @@
-{ lib, buildGoModule, fetchFromGitHub, kubernetes-helm }:
+{ lib
+, pkgs
```

Comment:
we don't pass pkgs for reasons. iirc mainly so that `.override` works properly.

just add `fuse` here and use it in `buildInputs`.